### ISSUE TITLE
请教一下，优化前后模型如何保存

### ISSUE THREAD

**Issue by MiXianif** [Muyun99/PaddleSlim_demo#1]
from __future__ import print_function
import paddle
from paddle.vision.models import mobilenet_v1
net = mobilenet_v1(pretrained=False)
paddle.summary(net, (1, 3, 32, 32))

import paddle.vision.transforms as T
transform = T.Compose([
                    T.Transpose(),
                    T.Normalize([127.5], [127.5])
                ])
train_dataset = paddle.vision.datasets.Cifar10(mode="train", backend="cv2",transform=transform)
val_dataset = paddle.vision.datasets.Cifar10(mode="test", backend="cv2",transform=transform)

# print(f'train samples count: {len(train_dataset)}')
# print(f'val samples count: {len(val_dataset)}')
# for data in train_dataset:
#     print(f'image shape: {data[0].shape}; label: {data[1]}')
#     break

from paddle.static import InputSpec as Input
optimizer = paddle.optimizer.Momentum(
        learning_rate=0.1,
        parameters=net.parameters())

inputs = [Input([None, 3, 32, 32], 'float32', name='image')]
labels = [Input([None, 1], 'int64', name='label')]

model = paddle.Model(net, inputs, labels)

model.prepare(
        optimizer,
        paddle.nn.CrossEntropyLoss(),
        paddle.metric.Accuracy(topk=(1, 5)))

model.fit(train_dataset, epochs=2, batch_size=128, verbose=1)
result = model.evaluate(val_dataset,batch_size=128, log_freq=10, verbose=0)
paddle.save(net.state_dict(), "./runs/FPGMFilterPruner/model.pdparams")

from paddleslim.dygraph import L1NormFilterPruner, FPGMFilterPruner
pruner = FPGMFilterPruner(net, [1, 3, 32, 32], opt=optimizer)

def eval_fn():
    result = model.evaluate(
        val_dataset,
        batch_size=128, verbose=0)
    return result['acc_top1']

pruner.sensitive(eval_func=eval_fn, sen_file="./sen.pickle")

from paddleslim.analysis import dygraph_flops
flops = dygraph_flops(net, [1, 3, 32, 32])
print(f"FLOPs before pruning: {flops}")

plan = pruner.sensitive_prune(0.4, skip_vars=["conv2d_26.w"])
paddle.save(net.state_dict(), "./runs/FPGMFilterPruner/pruning.pdparams")

flops = dygraph_flops(net, [1, 3, 32, 32])
print(f"FLOPs after pruning: {flops}")
print(f"Pruned FLOPs: {round(plan.pruned_flops*100, 2)}%")

result = model.evaluate(val_dataset,batch_size=128, log_freq=10, verbose=0)
print(f"before fine-tuning: {result}")

model.fit(train_dataset, epochs=2, batch_size=128, verbose=1)
result = model.evaluate(val_dataset,batch_size=128, log_freq=10, verbose=0)
print(f"after fine-tuning: {result}")
paddle.save(net.state_dict(), "./runs/FPGMFilterPruner/pruned.pdparams")

paddle.summary(net, (1, 3, 32, 32))

训练后的模型16630kb，剪枝后的模型35618kb，为什么剪枝后的模型比剪枝前的还大呢
训练前的paddle.summary：
=================================================================================
Total params: 4,253,864
Trainable params: 4,231,976
Non-trainable params: 21,888
---------------------------------------------------------------------------------
Input size (MB): 0.01
Forward/backward pass size (MB): 2.79
Params size (MB): 16.23
Estimated Total Size (MB): 19.03
---------------------------------------------------------------------------------

剪枝后的paddle.summary：
=================================================================================
Total params: 1,720,581
Trainable params: 1,705,127
Non-trainable params: 15,454
---------------------------------------------------------------------------------
Input size (MB): 0.01
Forward/backward pass size (MB): 2.14
Params size (MB): 6.56
Estimated Total Size (MB): 8.72
